deploy/operator.yaml

```diff
@@ -24,7 +23,9 @@ spec:
         imagePullPolicy: Always
         env:
         - name: WATCH_NAMESPACE
-          value: ""
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
```

Comment:
This is a change from what we had before. Previously, a default installation would watch all namespaces. Now, it watches only the same namespace the operator is deployed.